pkg/scheduler/framework/v1alpha1/context_test.go

```diff
@@ -64,3 +64,11 @@ func TestPluginContextClone(t *testing.T) {
 		t.Errorf("cloned copy should not change, got %q, expected %q", v.data, data1)
 	}
 }
+
+func TestPluginContextCloneNil(t *testing.T) {
```

Comment:
Ideally, this can be merged into `TestPluginContextClone` in a Golang [subtest](https://blog.golang.org/subtests) manner.

Reply:
I feel the boilerplate for doing will outweigh its benefit.

Reply:
Sure, it's just nits.